### ISSUE TITLE
database_cleaner 1.6.2 not installing on Windows: Errno::EACCES: Permission denied @ rb_file_s_symlink - (examples/../Gemfile

### ISSUE THREAD

**Issue by soundasleep** [DatabaseCleaner/database_cleaner#512]
When trying to use the latest `database_cleaner` gem:

```
Fetching database_cleaner 1.6.2
Installing database_cleaner 1.6.2
Using unf_ext 0.0.7.4 (x64-mingw32)
...
Using cucumber-core 1.5.0
Errno::EACCES: Permission denied @ rb_file_s_symlink - (examples/../Gemfile,
C:/Ruby/lib/ruby/gems/2.3.0/gems/database_cleaner-1.6.2/examples/Gemfile)
An error occurred while installing database_cleaner (1.6.2), and Bundler cannot
continue.
Make sure that `gem install database_cleaner -v '1.6.2'` succeeds before
bundling.
```

However if I change the version to `1.6.1` it works fine:

```
Fetching database_cleaner 1.6.1
Installing database_cleaner 1.6.1
...
Bundle complete! 37 Gemfile dependencies, 126 gems now installed.
```

Any ideas?

**Comment by etagwerker** [comment]
@soundasleep This should be fixed in `master` by #508. 

Could you test with `master` and confirm that it works?

**Comment by etagwerker** [comment]
Closing this because it's a duplicate of #506.
